src/Features/ERDDAP/Platform/Observations/CurrentConditions/wind.tsx

```diff
@@ -57,7 +54,7 @@ export const WindCard: React.SFC<Props> = ({ datasets, unit_system }) => {
   if (gust.length > 0) {
     gustTitle =
       " gusting to " +
-      round(data_converter.convertTo(gust[0].readings[gust[0].readings.length - 1].reading, unit_system) as number, 1) +
+      round(data_converter.convertToNumber(gust[0].readings[gust[0].readings.length - 1].reading, unit_system), 1) +
```

Comment:
Codacy found an issue: [Identifier 'data_converter' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6089628)